src/commands/daily.ts

```
@@ -120,7 +121,7 @@ export const dailyCommand = define({
 					'right',
 					'right',
 				],
-				dateFormatter: (dateStr: string) => formatDateCompact(dateStr, ctx.values.timezone),
+				dateFormatter: (dateStr: string) => formatDateCompact(dateStr, ctx.values.timezone, ctx.values.locale),
```

ESLint

[error] Unsafe argument of type `any` assigned to a parameter of type `string | undefined`.

(ts/no-unsafe-argument)